src/utils/dataTestIds.ts

```diff
@@ -105,8 +105,15 @@ export const dataTestId = {
   },
   footer: {
     aboutLink: 'about-link',
+    aboutSikt: 'about-sikt',
     availabilityStatement: 'availability-statement',
+    becomeCustomer: 'become-customer',
+    helpPage: 'help_page',
```

Comment:
samme naming convention som resten
```suggestion
    helpPage: 'help-page',
```